components/dashboard/src/App.tsx

```diff
@@ -298,6 +299,7 @@ function App() {
                 <Route path="/from-referrer" exact component={FromReferrer} />
 
                 <Route path="/admin/users" component={UserSearch} />
+                <Route path="/admin/teams" component={TeamsSearch} />
```

Comment:
Ah, I actually put it under `Users` because I thought Users and Teams group nicely as they're closely related?

Reply:
Here's what I had in mind:
1. Most of these are closely related except for Settings, but Teams & Projects have a stronger connection as this feature shipped as one piece[[1](https://www.gitpod.io/changelog/introducing-teams-&-projects)].
2. We've been deliberately using the term _Users_ here instead of _Members_ to include all users across all teams in the instance which creates a small disconnection from Teams & Projects hence the suggestion to move _Teams_ a bit downwards and closer to _Projects_.
3. Teams is also a larger container as a concept that includes projects, members, billing, etc. Cross-linking relevant FigJam file[[2](https://www.figma.com/file/V2p8Rd7swkV3swykNZIHGv/Projects---Prebuilds---Teams-(Brainstorm)?node-id=0%3A1)] (screenshot below) while discussing the introduction of Teams & Projects for more context.

| Early draft for the entity relationship diagram for the dashboard |
|-|
| ![image](https://user-images.githubusercontent.com/120486/155368431-e21758ac-959a-4575-98ae-9bdaeb1c640e.png) |

However, no need to hold this back because of this, @laushinka please move forward with any option that looks better to you. It's a small change with little to no negative impact, any option will suffice and we can easily change it later if needed. Your call. 🏀

Reply:
Ah that's right, I forgot that "Teams & Projects" was how we packaged it. It makes sense, will do the change :)